docs/designingProcesses/BasicNodes.md

```diff
@@ -9,94 +17,79 @@ Records from the `source` which meet filter's condition go to the `true sink`, a
 
 Records from the `source` which meets condition go to the `blue sink`, and others are filtered out. 
 
-### Parameters
 ![filter window](../img/filter_window.png)
-There are two parameters: `Expression`, and `Disabled`.
-
-Expression is written in SpEL and it is evaluated for each record to logical value. Outgoing data flow depends on expression result.
-
-When the node is Disabled, expression isn't evaluated, and always returns `true`.         
-
-### Flow
-Filter has one input and one or two outputs. 
-
-If there is one output, only outgoing pipe is named `true`, and each record which expression evaluates to true passes. Otherwise record is gone.
-
-If there are two outputs, one pipe is named `true` and another `false`. Each record which expression evaluates to `true` goes to `true` pipe, and other record goes to `false` pipe.  
-
-
-
-# Split 
```

Comment:
I wonder, maybe it's not bad to have a mention that split enables parallel processing of many branches?